test/unit/sql/select.test.js

```diff
@@ -83,7 +83,7 @@ describe(Support.getTestDialectTeaser('SQL'), function() {
       });
     });
 
-    it('include (left outer join)', function () {
+    it.skip('include (left outer join)', function () {
```

Comment:
Or well, actually not https://github.com/sequelize/sequelize/pull/3780/files?diff=unified

One  of those pertains to quoteIdentifers, so might be worth to keep that. Not that I like quoteIdentifers but I have a feeling several people rely on it